### PR TITLE
copyedits, Evaluating EC thru end

### PR DESCRIPTION
**Pull request by RacheltheEditor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1950,14 +1950,13 @@ to taxes on equity compensation:
...
-- In determining the value of equity, you have to figure out (or, more accurately, make
-  educated guesses about) several things.
-  We can start measuring the potential value of equity by doing our best to determine the
-  following:
+Now that you’re familiar with the [types of equity](#how-equity-is-granted) and how they
+[relate to taxes](#taxes-on-equity-compensation), you have many of the tools you need to
+evaluate an offer that includes equity compensation, or to evaluate equity you currently
+have in a company.
+
+- To measure the potential value of equity, you have to determine (or, more accurately, make
```
Comment (jlevy) [inline README.md]: Tightening: "you have to determine or make educated guesses about several things"

File: README.md [hunk 2]
```diff
@@ -1971,93 +1970,93 @@ includes equity compensation, or to evaluate equity you currently have in a comp
...
+  the questions a company should reasonably be expected to answer about offers they make.
 
 ### Questions to ask about offers
 
-🔹 It’s helpful to ask questions after you get an offer.
-Here are a few especially important ones especially relevant to startups and other private
-companies:
+🔹 It’s important to ask questions when you get an offer that includes any kind of equity.
+In addition to learning what you need to know to evaluate your offer, this process can you
+help you evaluate the company making the offer as well.
```
Comment (jlevy) [inline README.md]: Tightening: elide "as well"

File: README.md [hunk 3]
```diff
@@ -1971,93 +1970,93 @@ includes equity compensation, or to evaluate equity you currently have in a comp
...
 
-After you have the information, you’ll be able to consider the
-[possible scenarios](#common-scenarios) of when you might want to exercise options.
+When you have this information, you’ll be able to consider the
+[possible scenarios](#common-scenarios) of when you might want to exercise your options.
 
-🔹 If you’re considering working for a startup—that is, the company is so young the
-business has a lot of risk still—there are further questions to ask in order to assess the
-state of the company’s business.
+🔹 If you’re considering working for a startup, there are further questions to ask in
```
Comment (jlevy) [inline README.md]: 👍 on removing previously covered definition

File: README.md [hunk 4]
```diff
@@ -2088,35 +2087,37 @@ like this.
...
+    - 🔸 Recall that the [window is likely to close](#how-options-expire) soon after you leave a
+      company, often 90 days after termination.
 - **Wait until acquisition**: If the company is acquired for a large multiple of the exercise
   price, you may then use your options to buy valuable stock.
   However, as discussed, your shares could be worth next to nothing unless the sale price
-  exceeds the liquidation overhang, since preferred stock is paid out first.
-- 🔸 **Secondary market**: As [discussed](#what-is-your-stock-worth), in some cases it’s possible
-  to exercise and sell the stock in a private company directly to a private party.
+  exceeds the liquidation overhang.
+- 🔸 **Secondary market**: As [discussed](#can-you-sell-private-stock), in some cases it’s
```
Comment (jlevy) [inline README.md]: 👍

File: README.md [hunk 5]
```diff
@@ -2088,35 +2087,37 @@ like this.
...
-  combinations of them, such as exercising and then selling a portion to pay taxes.
-  In addition, there are a few funds or individual investors who may be able to front you
-  the cash to exercise or pay taxes in return for an agreement to share profits.
+
+🔹 Note that some of these scenarios may require significant cash up front, so it makes
+sense to do the math early.
+If you are in a tight spot, where you may lose valuable options altogether because you
+don’t have the cash to exercise, it’s worth exploring each of the scenarios above, or
+combinations of them, such as exercising and then selling a portion to pay taxes.
+In addition, there are a few funds and individual investors who may be able to front you
```
Comment (jlevy) [inline README.md]:
```
[few funds](#can-you-sell-private-stock)
```

File: README.md [hunk 6]
```diff
@@ -2133,28 +2134,28 @@ to devote a lot of your time and sanity to any full-time role;
 help yourself make sure that this is
 [what you want](https://hbr.org/2016/12/think-strategically-about-your-career-development).
 
-☝️ It’s perfectly natural to be anxious
+☝️ It’s perfectly natural to be
 [anxious](https://www.pon.harvard.edu/daily/negotiation-skills-daily/the-impact-of-anxiety-and-emotions-on-negotiations-how-to-avoid-misjudgment-in-negotiation-scenarios/)
-about negotiations, whether you’re going through this process for the first or the tenth
-time. There is a lot at stake, and it can be uncomfortable and stressful to ask for things
-that you need or want.
+about negotiations, whether you’re going through this process for the first or the tenth.
```
Comment (jlevy) [inline README.md]: first -> first time

File: README.md [hunk 7]
```diff
@@ -2164,27 +2165,29 @@ A Guide like this can’t give you personalized advice on what a reasonable offe
 that depends greatly on your skills, the marketplace of candidates, what other offers you
 have, what the company can pay, what other candidates the company has found, and the
 company’s needs.
-However, this section covers some basics of what to expect with offers, and tips on
+However, this section covers some basics of what to expect with offers, as well as tips on
 [negotiating an offer](https://hbr.org/2014/04/15-rules-for-negotiating-a-job-offer).
```
Comment (jlevy) [inline README.md]: (Not a copy edit but noting a reminder to myself to move this link later, under negotiation tips.)

File: README.md [hunk 8]
```diff
@@ -2164,27 +2165,29 @@ A Guide like this can’t give you personalized advice on what a reasonable offe
...
 in pay and opportunity span race and gender, with research focusing on
 [inequality in the U.S. workplace](http://www.pewresearch.org/fact-tank/2016/07/01/racial-gender-wage-gaps-persist-in-u-s-despite-some-progress/),
 [executive leadership](https://pr-paywatch-aflcio.pantheonsite.io/paywatch/company-pay-ratios)
-and its well documented
+and its well-documented
 [lack of diversity](http://fortune.com/2017/06/09/white-men-senior-executives-fortune-500-companies-diversity-data/),
 and the [technology industry](https://www.eeoc.gov/eeoc/statistics/reports/hightech/).
 [Gender bias](https://www.newyorker.com/science/maria-konnikova/lean-out-the-dangers-for-women-who-negotiate)
-in negotiation is also an issue, making many women feel uncomfortable negotiating.
+in negotiation itself is also an issue;
```
Comment (jlevy) [inline README.md]: Looks good, and look fine to me, so def let's keep. But a process note, these couple changes are edging beyond the assigned "copy-edits-only" tag into content changes, i.e. things that would benefit from content or subject matter (not copy edit) review.

File: README.md (hunk 2 above)
Comment (jlevy) [reply README.md]: Or perhaps clearer:

In addition to helping you learn the facts about the equity offer, the process of discussing these details can help you get a sense for the company's responsiveness and transparency.

File: README.md [hunk 9]
```diff
@@ -2235,19 +2239,19 @@ Things to look for in
   You know what this is now.
 - Bonus: Additional cash you’ll get on a regular basis, if the company has a plan for this.
 - [Signing bonus](https://www.investopedia.com/terms/s/signing-bonus.asp): Cash you get just for
-  signing. Note signing bonuses usually have some strings attached—for example you will have
-  to pay it back if you leave the company within 12 or 24 months.
+  signing. (Signing bonuses usually have some contingencies—for example, you could have to
```
Comment (jlevy) [inline README.md]: What was wrong with "strings attached"? It is an informal term, but for that reason safer to use when describing things informally.

*Contingency* has other technical meanings so can be confused (for example contingency fees are a different thing), so this change is likely not a "copy-edit-only" as it has some danger.

File: README.md [hunk 10]
```diff
@@ -2235,19 +2239,19 @@ Things to look for in
...
+  pay it back if you leave the company within 12 or 24 months.)
```
Comment (jlevy) [inline README.md]: it -> the bonus

File: README.md [hunk 11]
```diff
@@ -2420,17 +2423,17 @@ Some notes on the negotiation process itself:
   This can put the company in a difficult position (they may have declined another key
   candidate based on your acceptance), and may hurt your reputation in unexpected ways
   later.
-- Robby Grossman, a VP at Wistia, gives
-  [a good overview](http://rob.by/2013/negotiating-your-startup-job-offer/) of equity
-  compensation and negotiation suggestions.
+- Robby Grossman, a VP at Wistia, gives a
```
Comment (jlevy) [inline README.md]: 👍 on keeping links consistently on the noun sans article.

File: README.md (hunk 1 above)
Comment (RacheltheEditor) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (RacheltheEditor) [reply README.md]: Fixed. (Replaced with suggested text.)

File: README.md (hunk 5 above)
Comment (RacheltheEditor) [reply README.md]: Fixed.

File: README.md (hunk 6 above)
Comment (RacheltheEditor) [reply README.md]: Fixed!

File: README.md (hunk 7 above)
Comment (RacheltheEditor) [reply README.md]: 👍

File: README.md (hunk 8 above)
Comment (RacheltheEditor) [reply README.md]: Yes, a content change! Felt it was an urgent correction, but totally get it. Thanks for stetting.

File: README.md (hunk 9 above)
Comment (RacheltheEditor) [reply README.md]: I did not anticipate the term's technical reference. Nothing wrong with "strings attached," just wanted to tighten the sentence. Fixed!

File: README.md (hunk 10 above)
Comment (RacheltheEditor) [reply README.md]: Fixed.

File: README.md (hunk 11 above)
Comment (RacheltheEditor) [reply README.md]: 👍

File: README.md (hunk 7 above)
Comment (jlevy) [reply README.md]: Now done.

File: README.md (hunk 8 above)
Comment (jlevy) [reply README.md]: Yeah. @Hopeopolis do you have any notes on the changes on (only) just these couple changes in this subsection? We merged this along with the copy edits but would be good to get your review as well as this was your material.

File: README.md [hunk 12]
```diff
@@ -2386,31 +2391,31 @@ Some points on negotiating compensation:
...
+A few notes on the negotiation process itself:
 
 - 🔹 Although offer letters have expirations, it’s often possible to negotiate more time if
   you need it. How much flexibility depends on the situation.
-  Some people will go so far as to say “exploding job offers”
-  [are a bad practice](https://www.huffingtonpost.com/adam-grant/its-time-to-eliminate-exp_b_4594222.html)
-  or
+  Some have criticized “exploding job offers” as a
+  [bad practice](https://www.huffingtonpost.com/adam-grant/its-time-to-eliminate-exp_b_4594222.html)
+  that
```
Comment (jlevy) [inline README.md]: Tense now wrong below.

make -> makes

**Pull request merged**